src/utils/court-utils.js

```diff
@@ -54,16 +42,30 @@ export function getExpectedCurrentTermId(now, terms, termDuration) {
   return currentTermId
 }
 
-export function getTermStartAndEndTime(termId, terms, termDuration) {
-  let [termStartTime, termEndTime] = [0, 0]
+export function getTermStartTime(termId, { terms, termDuration }) {
+  let termStartTime = 0
 
   if (terms.length > 0) {
     const firstTermStartTime = terms[0].startTime
     termStartTime = termId * termDuration + firstTermStartTime
-    termEndTime = termStartTime + (termDuration - 1)
   }
 
-  return { termStartTime, termEndTime }
+  return termStartTime
+}
+
+export function getTermEndTime(termId, { terms, termDuration }) {
+  let termEndTime = 0
+
+  const termStartTime = getTermStartTime(termId, { terms, termDuration })
+  termEndTime = termStartTime + (termDuration - 1)
+
+  return termEndTime
```

Comment:
Only comment for the above is whether we should use a default of `0`; that doesn't seem to be very useful and we should perhaps error in cases where we can reach them (only in `getTermStartTime()` it seems?)

Reply:
The default `0` is because when loading the court, the court config may haven't loaded yet. 

> we should perhaps error in cases where we can reach them

What do you mean exactly by this?

Reply:
> we should perhaps error in cases where we can reach them

I'm wondering if we should not instead expect the user to make sure that they're providing valid inputs to this function

If the court's config or current term has not been loaded yet, I think avoiding a call to this function is better than calling and having to handle the `0` default.

Reply:
Good point, updated accordingly.